tests/test_vacuum_importance_sampling.py

```diff
@@ -12,8 +12,8 @@
 import jax
 import numpy as np
 import pytest
+import test_ligands
```

Comment:
Why did this change?

Reply:
The current `from tests import test_ligands` fails when the package is installed without `-e`:

```
ModuleNotFoundError: No module named 'tests'
```

To be honest I'm not quite sure how this worked before, but this change seems correct to me; `tests` is not a module with an `__init__.py` file, so the absolute import doesn't make sense to me.

Reply:
Good call, I went ahead and converted `tests` to a module in b86e0f7. This allows using relative import syntax to more clearly distinguish local from package imports.

Reply:
>To be honest I'm not quite sure how this worked before

I think this had assumed path/to/timemachine was in python path -- good to fix this!

Reply:
Strange, adding `__init__.py` to `tests` in b86e0f7768a463dfb3ac480884f0d989ceb95073 seems to have broken the build. I'm investigating why; seems good to understand rather than just reverting the change.

Reply:
> Strange, adding __init__.py to tests in b86e0f7 seems to have broken the build.

I think this is due to the ambiguity around whether to import from the installed package or from the source directory when we run tests from the repo root directory. It seems like the structure we currently have, with tests next to the module root directory, is discouraged for this reason (e.g. [here](https://stackoverflow.com/a/50796779/1427284)).

The recommendation seems to be to move the module root under `src` or similar ([example](https://blog.ionelmc.ro/2014/05/25/python-packaging/#the-structure)). Maybe we could consider something like this for "deboggling" week? For now, I propose to revert the changes making `tests` a module.

Reply:
Created #679 to track this.